protocol/streaming/full_node_streaming_manager.go

```diff
@@ -405,51 +405,32 @@ func (sm *FullNodeStreamingManagerImpl) SendSubaccountUpdate(
 			SubaccountUpdate: &subaccountUpdate,
 		},
 	}
-	sm.stageFinalizeBlockEvent(
+	sm.finalizeBlockStager.StageFinalizeBlockEvent(
 		ctx,
 		sm.cdc.MustMarshal(&stagedEvent),
 	)
```

Comment:
meganit: the Getter returns `[]T` so let's also pass in `T` instead of `[]byte` for consistency